File: front-end/admin-web/src/constants/hub.constants.js
import { env } from 'env'

export const HubConnectionConstants = {
  ORDER_HUB: `${env.REACT_APP_ROOT_DOMAIN}/orderHub`
}

export const OrderHubConstants = {
  RECEIVE_ORDER: 'ReceiveOrder',
  UPDATE_STATUS_BY_CUSTOMER: 'UpdateStatusByCustomer',
  UPDATE_ORDER_BY_CUSTOMER: 'UpdateOrderByCustomer'
}

File: back-end/eShopping.Common/Constants/HubConstants.cs
using eShopping.Domain.Enums;
using System.Collections.Generic;

namespace eShopping.Common.Constants
{
    public class HubConnectionConstants
    {
        public const string OrderHub = "/orderHub";
    }

    public class OrderHubConstants
    {
        public const string RECEIVE_ORDER = "ReceiveOrder";
        public const string UPDATE_STATUS_BY_CUSTOMER = "UpdateStatusByCustomer";
        public const string UPDATE_ORDER_BY_CUSTOMER = "UpdateOrderByCustomer";

        public const string UPDATE_STATUS_BY_STAFF = "UpdateStatusByStaff";
        public const string UPDATE_ORDER_BY_STAFF = "UpdateOrderByStaff";
    }
}